apps/discordbot/src/discord/command/abbreviation-command/abbreviation-command.service.ts

```diff
@@ -0,0 +1,32 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { Injectable } from '@nestjs/common';
+import {
+  CommandInteraction,
+  CacheType,
+  ChatInputCommandInteraction,
+} from 'discord.js';
+import { Command } from '../command';
+
+@Injectable()
+export class AbbreviationCommandService extends Command {
+  public COMMAND_NAME = 'abbreviation';
+
+  public setup(): SlashCommandBuilder {
+    return new SlashCommandBuilder()
+      .setName('abbreviation')
```

Comment:
Can we retrieve this name from the class variable defined above?

Reply:
I think I implented your suggestion, if I understood it correctly. And in the toneindicator command as well.